gradle.properties

```diff
@@ -2,20 +2,21 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx1G
 org.gradle.warning.mode=all
 # Check these on https://modmuss50.me/fabric.html
-minecraft_version=1.18
-yarn_mappings=1.18+build.1
-loader_version=0.12.8
+minecraft_version=1.19
+yarn_mappings=1.19+build.4
+loader_version=0.14.8
 #Fabric api
-fabric_version=0.43.1+1.18
-loom_version=0.10-SNAPSHOT
+fabric_version=0.57.0+1.19
+loom_version=0.12-SNAPSHOT
 # Mod Properties
-modVersion=1.5.1
+modVersion=1.6.0
```

Comment:
This should still be 1.5.1, no changes have been made other than a simple port so no version bump is needed.

Reply:
Done in the latest commit.